bodhi/server/util.py

```diff
@@ -343,6 +343,20 @@ def status2html(context, status):
     return "<span class='label label-%s'>%s</span>" % (cls, status)
 
 
+def ci_status2html(context, status):
+    cls = {
+        'Ignored': 'success',
+        'Running': 'warning',
+        'Passed': 'success',
+        'Failed': 'danger',
+        'Queued': 'info',
+        'None': 'primary',
+    }[str(status)]
+    if status is None:
+        status = 'info awaited'
```

Comment:
If we're going to set historic updates to "Ignored", we should probably not allow status to be None.